builder/builder.go

```diff
@@ -465,10 +461,14 @@ func (b *Builder) copyImageTemplate(configFile, templateFile string) error {
 	if err == nil && len(mixBundles) > 0 {        // check if set is not empty
 		var bundles []string
 		for _, bundle := range mixBundles {
+			// Ignore additional kernel bundles
+			if strings.HasPrefix(bundle.Name, "kernel-") {
```

Comment:
So the functional difference w.r.t to how ister worked is that,
If the user had built a mix with a different kernel like "kernel-lts", and tried to build an image with default "release-image-config.yaml", the image that gets created will have the default "kernel-native" and the "kernel-lts" would be ignored.